Tools/LyTestTools/ly_test_tools/launchers/platforms/base.py

```diff
@@ -153,7 +153,6 @@ def teardown(self):
         :return: None
         """
         self.workspace.asset_processor.stop()
-        self.save_project_log_files()
```

Comment:
This will mean that tests which directly launch the Editor or Game.Launcher no longer persist logs, and only tests using editor_test.py will persist.

Reply:
The contents of this folder may still be relevant to a test using editor_test.py, such as instances where an application's logs or other artifacts still end up in the project-logs location. This could involve bugs or misconfiguration.

Reply:
Parallel editor tests will create their own custom project log location where its respective editor log will be saved. If this default saving of logs is kept, parallel editor tests will continue to save incorrect logs. This means that the individual tests will need to be responsible for saving their own logs using this interface.